R/Mexico.R

```diff
@@ -99,6 +99,14 @@ Mexico <- R6::R6Class("Mexico",
           self$verbose
         )
 
+        dat <- suppressWarnings(
```

Comment:
shouldn't this go in data cleaning?

Reply:
Yeah I think thats the best place for it